.travis.yml

```
@@ -77,10 +77,7 @@ matrix:
     stage: test
 
 before_install:
-    - sudo mkdir -p /var/log/celery
-    - sudo mkdir -p /var/run/celery
-    - sudo chown travis /var/log/celery
-    - sudo chown travis /var/run/celery
+    - sudo install -d -g travis /var/log/celery /var/run/celery
```

```suggestion
    - sudo install --directory --owner=travis /var/log/celery /var/run/celery
```

whoops, we want to change the owner not the group